helm/oauth2-proxy/README.md

```diff
@@ -357,3 +358,6 @@ config:
     whitelist_domains = [ ".domain.com", ".example.io"]
     provider = "google"
 ```
+
+## Route requests to sidecar container
+You can route requests to a sidecar container first by setting the `service.targetPort` variable; by default, the service's `targetPort` value equals to the `httpSchema`'s.
```

Comment:
The possible values for the `targetPort` field of a Kubernetes Service can be either a port number or the name of a port defined in the pod.

Reply:
Changed in [a423d1d](https://github.com/oauth2-proxy/manifests/pull/270/commits/a423d1d151aa387935ee105b7845f81d47d9564f)